scripts/build.py

```diff
@@ -181,6 +181,91 @@ def _ensure_files_exist(filepaths):
             raise Exception('File %s does not exist.' % filepath)
 
 
+def _ensure_fonts_exist(filepaths):
+    """Ensures that fonts exists at the given filepaths.
+
+    Args:
+        filepaths: list(str). Wildcard paths to fonts.
+
+    Raises:
+        Exception: Raised if one or more of the files does not exist.
+    """
+    for font_wildcard in filepaths:
+        font_paths = glob.glob(font_wildcard)
+        _ensure_files_exist(font_paths)
+
+
+def get_file_count(directory_path):
+    """Count total number of file directory, subtracting ignored files.
+
+    Args:
+        directory_path: str. Directory to be walked.
+
+    Returns:
+        total_file_count: int. Total number of files minus ignored files.
+    """
+    total_file_count = 0
+    #pylint: disable=unused-variable
+    for root, dirs, files in os.walk(directory_path):
+        #pylint: enable=unused-variable
```

Comment:
You can use _ for unused variables I think.

Reply:
If both `root` and `dirs` are unused, would that make things seem confusing?

Reply:
Nope, it's fine. Use _